contrib/release.sh

```
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -e
+set -o pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)